files/templates/scrapeconfigs/_apiserver.yaml

```diff
@@ -1,10 +1,15 @@
 [[- define "_apiserver" -]]
 [[- if ne .ClusterType "management_cluster" ]]
     api_server: https://[[ .APIServerURL ]]
+[[- if eq .AuthenticationType "token" ]]
```

Comment:
Add a new field named authentication type that explains if the tls config needs to use a token or a cert/key pair